allennlp/sanity_checks/task_checklists/question_answering_suite.py

```diff
@@ -0,0 +1,207 @@
+from typing import Optional, Iterable, Tuple
+import itertools
+import sys
+from allennlp.sanity_checks.task_checklists.task_suite import TaskSuite
+from allennlp.sanity_checks.task_checklists import utils
+from checklist.test_suite import TestSuite
+from checklist.test_types import MFT
+from checklist.perturb import Perturb
+import numpy as np
+from overrides import overrides
```

Comment:
Nitpick about organizing imports

```suggestion

from checklist.test_suite import TestSuite
from checklist.test_types import MFT
from checklist.perturb import Perturb
import numpy as np
from overrides import overrides

from allennlp.sanity_checks.task_checklists.task_suite import TaskSuite
from allennlp.sanity_checks.task_checklists import utils
```

Reply:
Ah, yes. Wasn't there something in our formatters that did this? Am I misremembering it?